kie-wb-tests/kie-wb-tests-rest/src/test/java/org/kie/wb/test/rest/RestTestBase.java

```diff
@@ -28,9 +33,11 @@
 
 public abstract class RestTestBase {
 
-    private static final String URL = System.getProperty("kie.wb.url", "http://localhost:8080/kie-wb");
+    protected static final String URL = System.getProperty("kie.wb.url", "http://localhost:8080/kie-wb");
     protected static final String USER_ID = System.getProperty("kie.wb.user.name", User.REST_ALL.getUserName());
-    private static final String PASSWORD = System.getProperty("kie.wb.user.password", User.REST_ALL.getPassword());
+    protected static final String PASSWORD = System.getProperty("kie.wb.user.password", User.REST_ALL.getPassword());
+
+    protected static final String GIT_URL_REMOTE = "https://github.com/droolsjbpm/jbpm-playground.git";
```

Comment:
I was thinking about some other solution and nothing came to my mind. But a git repository on a local file system sounds good. I will try to implement it using JGit.